server/handlers/notifications.go

```diff
@@ -0,0 +1,96 @@
+package handlers
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-msteams/server/metrics"
+	"github.com/mattermost/mattermost-plugin-msteams/server/msteams/clientmodels"
+)
+
+func (ah *ActivityHandler) handleCreatedActivityNotification(msg *clientmodels.Message, chat *clientmodels.Chat) string {
+	botUserID := ah.plugin.GetBotUserID()
+	post, _, _ := ah.msgToPost("", botUserID, msg, chat, false, []string{})
+
+	if chat == nil {
+		// We're only going to support notifications from chats for now.
+		return metrics.DiscardedReasonChannelNotificationsUnsupported
+	}
+
+	for _, member := range chat.Members {
+		// Don't notify senders about their own posts.
+		if member.UserID == msg.UserID {
```

Comment:
nit: Should we have an exception for self messaging?

Reply:
I'd advocate for silencing self-messaging here -- maaaaybe useful for testing, but super annoying if Teams users use their own chats like we sometimes do on Mattermost as a scratch pad.